packages/tupaia-web/src/api/queries/useLandingPage.ts

```diff
@@ -5,14 +5,13 @@
  */
 
 import { useQuery } from 'react-query';
-import { useLocation } from 'react-router';
+import { useParams } from 'react-router';
 import { get } from '../api';
 import { SingleLandingPage } from '../../types';
 
-export const useLandingPage = (urlSegment?: string) => {
-  const { pathname } = useLocation();
```

Comment:
I think this part here might still be needed - in some views, (e.g. in a modal route) the `landingPageUrlSegment` param isn't accessible, so it has to be grabbed from the pathname